tcf_website/static/base/base.css

```diff
@@ -79,15 +64,16 @@ footer > .row {
 .btn.btn-primary:hover,
 .btn.btn-outline-primary:hover {
   color: white;
-  background-color: #d1420d;
-  border-color: #d1420d;
+  background-color: #AC451E;
+  border-color: #AC451E;
 }
 
 .btn:not(:disabled):not(.disabled).active.focus,
 .btn:not(:disabled):not(.disabled).active:focus {
   box-shadow: none;
 }
 
+/* For forms */
 .form-control {
   background-color: var(--background-color);
```

Comment:
Could you modify either this background color or the `form-control:disabled` background color? The problem is the two colors are extremely similar and it's difficult to tell when a field is disabled. Another option (and/or) is to add `cursor: not-allowed` to the disabled fields

Reply:
Edit: my styles were out of date so I didn't see that the text gets greyed out. That's already pretty good, but the problem is the placeholder text in the `text-area` is greyed out too so it looks the same as the disabled `select` fields

Reply:
So you think I should change the background color of disabled fields? Can do, but I'll need to add another shade of grey to the style guide hmm

Reply:
done